src/avfoundation.cs

```diff
@@ -14343,4 +14841,828 @@ public enum AVCaptureCenterStageControlMode : long {
+		string GetIdentifier (AVPlayerPlaybackCoordinator coordinator, AVPlayerItem playerItem);
+	}
+
+	[TV (15,0), NoWatch, Mac (12,0), iOS (15,0), MacCatalyst (15,0)]
+	[BaseType (typeof (AVPlaybackCoordinator))]
+	[DisableDefaultCtor]
+	interface AVPlayerPlaybackCoordinator
+	{
+		[NullAllowed, Export ("player", ArgumentSemantic.Weak)]
+		AVPlayer Player { get; }
+
+		[Wrap ("WeakDelegate")]
+		[NullAllowed]
+		IAVPlayerPlaybackCoordinatorDelegate Delegate { get; set; }
+
+		[NullAllowed, Export ("delegate", ArgumentSemantic.Weak)]
+		NSObject WeakDelegate { get; set; }
+	}
+
+	interface IAVAssetReaderCaptionValidationHandling {}
+
+	[NoWatch, NoTV, NoiOS, Mac (12,0), NoMacCatalyst]
+	[Protocol]
+	[BaseType (typeof(NSObject))]
+	interface AVAssetReaderCaptionValidationHandling
+	{
+		[Export ("captionAdaptor:didVendCaption:skippingUnsupportedSourceSyntaxElements:")]
+		void DidVendCaption (AVAssetReaderOutputCaptionAdaptor adaptor, AVCaption caption, string[] syntaxElements);
+	}
+
+	[NoWatch, NoTV, NoiOS, Mac (12,0), NoMacCatalyst]
+	[BaseType (typeof(NSObject))]
+	[DisableDefaultCtor]
+	interface AVAssetReaderOutputCaptionAdaptor
+	{
+		[Static]
+		[Export ("assetReaderOutputCaptionAdaptorWithAssetReaderTrackOutput:")]
+		AVAssetReaderOutputCaptionAdaptor Create (AVAssetReaderTrackOutput trackOutput);
+
+		[Export ("initWithAssetReaderTrackOutput:")]
+		IntPtr Constructor (AVAssetReaderTrackOutput trackOutput);
+
+		[Export ("assetReaderTrackOutput")]
+		AVAssetReaderTrackOutput AssetReaderTrackOutput { get; }
+
+		[NoMacCatalyst]
+		[Export ("nextCaptionGroup")]
+		[return: NullAllowed]
+		AVCaptionGroup GetNextCaptionGroup ();
+
+		[NoMacCatalyst]
+		[Export ("captionsNotPresentInPreviousGroupsInCaptionGroup:")]
+		AVCaption[] GetCaptionsNotPresent (AVCaptionGroup captionGroup);
+
+		// interface AVAssetReaderOutputCaptionAdaptor_AVAssetReaderCaptionValidation
+		[Wrap ("WeakValidationDelegate")]
+		[NullAllowed]
+		IAVAssetReaderCaptionValidationHandling ValidationDelegate { get; set; }
+
+		[NullAllowed, Export ("validationDelegate", ArgumentSemantic.Weak)]
+		NSObject WeakValidationDelegate { get; set; }
+	}
+
+	[NoWatch, NoTV, NoiOS, Mac (12,0), MacCatalyst (15,0)]
+	[BaseType (typeof(NSObject))]
+	[DisableDefaultCtor]
+	interface AVAssetWriterInputCaptionAdaptor
+	{
+		[Static]
+		[Export ("assetWriterInputCaptionAdaptorWithAssetWriterInput:")]
+		AVAssetWriterInputCaptionAdaptor Create (AVAssetWriterInput input);
+
+		[Export ("initWithAssetWriterInput:")]
+		IntPtr Constructor (AVAssetWriterInput input);
+
+		[Export ("assetWriterInput")]
+		AVAssetWriterInput AssetWriterInput { get; }
+
+		[NoMacCatalyst]
+		[Export ("appendCaption:")]
+		bool AppendCaption (AVCaption caption);
+
+		[NoMacCatalyst]
+		[Export ("appendCaptionGroup:")]
+		bool AppendCaptionGroup (AVCaptionGroup captionGroup);
+	}
+
+	[NoWatch, NoTV, NoiOS, Mac (12,0), NoMacCatalyst]
+	[BaseType (typeof(NSObject))]
```

Comment:
-> `[DisableDefaultCtor]`
there's no way to set captions or timeRange and they are not documented to allow `nil`